velox/docs/functions/spark/math.rst

```diff
@@ -96,6 +96,14 @@ Mathematical Functions
 
     Returns Euler's number raised to the power of ``x``.
 
+.. spark:function:: expm1(x) -> double
+
+    Returns Euler's number raised to the power of ``x``, minus 1, which is ``exp(x) - 1`` in math. This function expm1(x) is more accurate than ``std::exp(x) - 1`` when ``x`` is close zero.
```

Comment:
> is close zero.

is close to zero.

> std::exp(x) - 1

exp(x) - 1 since this is documentation of the SQL function, not its C++ equivalent.